ldelf/sub.mk

```diff
@@ -4,6 +4,8 @@ srcs-$(CFG_ARM64_$(sm)) += start_a64.S
 srcs-$(CFG_ARM32_$(sm)) += syscalls_a32.S
 srcs-$(CFG_ARM64_$(sm)) += syscalls_a64.S
 srcs-$(CFG_ARM64_$(sm)) += tlsdesc_rel_a64.S
+srcs-$(CFG_X86_64_$(sm)) += start_64.S
+srcs-$(CFG_X86_64_$(sm)) += syscalls_64.S
```

Comment:
Suggest to change to the following:
ifeq ($(X86_64),1)
srcs-y += start_64.S
srcs-y += syscalls_64.S
#endif